### PR TITLE
feat(VPagination): add a11y support

### PR DESCRIPTION
**Pull request by abnersajr**
## Description

This PR adds `aria-*` attributes improving accessibility of the pagination component.
Another feature is the component localization with translations to all available locations.

## Motivation and Context

It fixes the issue #9849 following all the instructions requested.

## How Has This Been Tested?

I manually tested and also the unit test was fixed to work with the new changes.

## Markup:

<details>

```vue
<template>
  <v-container>
    <v-row justify="center">
        <v-col cols="8">
          <v-container class="max-width">
            <v-pagination
              v-model="page"
              class="my-4"
              :length="15"
            ></v-pagination>
          </v-container>
        </v-col>
      </v-row>
  </v-container>
</template>

<script>
export default {
  data: () => ({
    page: 1,
  })
}
</script>

```
</details>

## Types of changes

- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.

### CODE REVIEW

File: packages/vuetify/src/locale/en.ts [hunk 1]
```diff
@@ -44,4 +44,13 @@ export default {
     am: 'AM',
     pm: 'PM',
   },
+  pagination: {
+    ariaLabel: {
+      wrapper: 'Pagination Navigation',
+      next: 'Next page',
+      previous: 'Previous page',
+      page: 'Goto Page {0}',
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/locale/en.ts]:
```suggestion
      page: 'Goto page {0}',
```

File: packages/vuetify/src/locale/en.ts [hunk 2]
```diff
@@ -44,4 +44,13 @@ export default {
     am: 'AM',
     pm: 'PM',
   },
+  pagination: {
+    ariaLabel: {
+      wrapper: 'Pagination Navigation',
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/locale/en.ts]:
```suggestion
      wrapper: 'Pagination navigation',
```

File: packages/vuetify/src/locale/en.ts [hunk 3]
```diff
@@ -44,4 +44,13 @@ export default {
...
+      next: 'Next page',
+      previous: 'Previous page',
+      page: 'Goto Page {0}',
+      currentPage: 'Current Page, Page {0}',
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/locale/en.ts]:
```suggestion
      currentPage: 'Current page: {0}',
```
This sounds better for me, applies for other languages too

File: packages/vuetify/src/locale/pl.ts [hunk 4]
```diff
@@ -44,4 +44,13 @@ export default {
     am: 'AM',
     pm: 'PM',
   },
+  pagination: {
+    ariaLabel: {
+      wrapper: 'Nawigacja paginacyjna',
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/locale/pl.ts]:
```suggestion
      wrapper: 'Paginacja',
```

File: packages/vuetify/src/components/VPagination/VPagination.ts [hunk 5]
```diff
@@ -179,22 +185,30 @@ export default mixins(Colorable, Themeable).extend({
...
+        }],
+        class: this.classes,
+      }, children)
+    },
   },
 
   render (h): VNode {
     const children = [
-      this.genIcon(h, this.$vuetify.rtl ? this.nextIcon : this.prevIcon, this.value <= 1, this.previous),
+      this.genIcon(h, this.$vuetify.rtl ? this.nextIcon : this.prevIcon, this.value <= 1, this.previous, this.$vuetify.lang.t('$vuetify.pagination.ariaLabel.previous')),
```
Comment (johnleider) [inline packages/vuetify/src/components/VPagination/VPagination.ts]: This should be a prop so the user can change it.

File: packages/vuetify/src/components/VPagination/VPagination.ts [hunk 6]
```diff
@@ -179,22 +185,30 @@ export default mixins(Colorable, Themeable).extend({
...
       this.genItems(h),
-      this.genIcon(h, this.$vuetify.rtl ? this.prevIcon : this.nextIcon, this.value >= this.length, this.next),
+      this.genIcon(h, this.$vuetify.rtl ? this.prevIcon : this.nextIcon, this.value >= this.length, this.next, this.$vuetify.lang.t('$vuetify.pagination.ariaLabel.next')),
```
Comment (johnleider) [inline packages/vuetify/src/components/VPagination/VPagination.ts]: This should be a prop so the user can change it.

File: packages/vuetify/src/components/VPagination/VPagination.ts [hunk 7]
```diff
@@ -179,22 +185,30 @@ export default mixins(Colorable, Themeable).extend({
...
-        modifiers: { quiet: true },
-        name: 'resize',
-        value: this.onResize,
-      }],
-      class: this.classes,
-    }, children)
+    return h('nav', {
+      attrs: {
+        role: 'navigation',
+        'aria-label': this.$vuetify.lang.t('$vuetify.pagination.ariaLabel.wrapper'),
```
Comment (johnleider) [inline packages/vuetify/src/components/VPagination/VPagination.ts]: This should be a prop so the user can change it.

**Comment by johnleider** [comment]
I've moved this to the `dev` branch since it's a feature.

**Comment by abnersajr** [comment]
@johnleider I think in case we decide to go as `props` I think this will lose the idea of be localized.
What I can consider is make the change through props as an alternative. And let the localized as default.
I would prefer make this change in another PR. Even it's nothing big, but to be separated the ideas.
What you think?

**Comment by jacekkarczmarczyk** [comment]
@abnersajr prop can have the default value pointing to the locales, this will give possibility to change the value globally (in locales) or per component (in prop). See for example https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VAlert/VAlert.ts#L44

**Comment by johnleider** [comment]
@abnersajr pulse check on this.

**Comment by abnersajr** [comment]
@johnleider sorry. Things are going crazy this year begin. I will solve it by tomorrow.👊

**Comment by abnersajr** [comment]
@johnleider @jacekkarczmarczyk all comments done.

**Comment by abnersajr** [comment]
I made a mess with the commits, I'm trying to solve this. In case someone have a suggestion please let me know.

**Comment by johnleider** [comment]
Is this non-breaking? If so, I believe we can get it in as a bug fix seeing as it pertains to a11y.

**Pull request merged**